From b89af27c4fd1e9ed9ea0fb4f1c53f07d1cc351d2 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Sun, 24 Jan 2016 13:16:54 +0100
Subject: [PATCH 2/2] Partially revert "Re-enabling code for QtOAuth"

This partially reverts commit 7b6937326ba2a4e4072692add38a4abd28bd0cd4
so that kbibtex is using system-qoauth instead of bundled one.
---
 src/networking/CMakeLists.txt         | 3 ---
 src/networking/zotero/oauthwizard.cpp | 2 +-
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/networking/CMakeLists.txt b/src/networking/CMakeLists.txt
index 8752f27..58a124b 100644
--- a/src/networking/CMakeLists.txt
+++ b/src/networking/CMakeLists.txt
@@ -42,8 +42,6 @@ if(NOT WIN32)
         zotero/groups.cpp
         zotero/tags.cpp
         zotero/tagmodel.cpp
-        # QOAuth for Qt5 from Git
-        ../3rdparty/qoauth/src/interface.cpp
         )
 endif(NOT WIN32)
 
@@ -88,7 +86,6 @@ if(UNITY_BUILD)
 endif(UNITY_BUILD)
 
 include_directories(
-    ${CMAKE_SOURCE_DIR}/src/3rdparty/qoauth/include
     ${CMAKE_BINARY_DIR}/src/config
     ${CMAKE_SOURCE_DIR}/src/config
     ${CMAKE_BINARY_DIR}/src/data
diff --git a/src/networking/zotero/oauthwizard.cpp b/src/networking/zotero/oauthwizard.cpp
index 1d9129e..91ff23b 100644
--- a/src/networking/zotero/oauthwizard.cpp
+++ b/src/networking/zotero/oauthwizard.cpp
@@ -34,7 +34,7 @@
 #include "internalnetworkaccessmanager.h"
 #include "logging_networking.h"
 
-#include <QtOAuth>
+#include <QtOAuth/QtOAuth>
 
 using namespace Zotero;
 
-- 
2.10.2

